lib/load-tasks.js

```diff
@@ -2,11 +2,19 @@
 const RunTask = require('runtask');
 const async = require('async');
 const cachedTasks = {};
+const path = require('path');
+const ClientkitTask = require('clientkit-task');
 
-module.exports = function(config, log, done) {
+const hooks = ['preInit', 'postInit', 'process', 'postProcess'];
```

Comment:
I'm not sure it makes sens to add these here since clientkit-core isn't aware of any tasks that are defined.  Can't we just add `preInit: ['clientkit-task']` in clientkit repo?